elements/lisk-p2p/src/validation.ts

```diff
@@ -158,11 +160,15 @@ export const checkProtocolVersionCompatibility = (
 	nodeInfo: P2PNodeInfo,
 ): boolean => {
 	if (!peerInfo.protocolVersion) {
-		return false;
+		try {
+			return isVersionGTE(peerInfo.version, nodeInfo.minVersion as string);
+		} catch (error) {
```

Comment:
If error is not being used it can be omitted here :)

Reply:
I think error cannot be ommited by the node.js definition =p

Reply:
Maybe `_`? hahaha. It's fine then